servant-persistent.cabal

```diff
@@ -47,37 +47,41 @@ library
         src
     exposed-modules:
         Config
+      , Init
       , Models
       , Api
       , Api.User
       , Logger
+      , DevelMain
     build-depends:
         base >= 4.9 && < 5.0
       , aeson
       , bytestring
+      , containers
+      , ekg
+      , ekg-core
+      , fast-logger
+      , foreign-store
```

Comment:
Most of the dependencies are just sorted alphabetically, but this one is real. We use it to persist state in GHCi between reloads.